classes/core/PKPRequest.inc.php

```diff
@@ -591,6 +591,18 @@ function &getUser() {
 	function getUserVar($key) {
 		$_this = PKPRequest::_checkThis();
 
+		if (is_a($this, 'Request')) {
```

Comment:
@asmecher : so I came across the error again. 

With `$_this`, when I navigate to API Key tab in the profile page, I get the error below:

`/index.php/publicknowledge/$$$call$$$/tab/user/profile-tab/api-profile?_=1497075388298 - Call to undefined method APIProfileForm::getRouter() in /<path-to-ojs>/lib/pkp/classes/core/PKPRequest.inc.php on line 596`